src/Valet/App.cs

```diff
@@ -74,6 +74,29 @@ public async Task<int> GetVersionAsync()
         return 0;
     }
 
+    public async Task CheckForUpdates()
```

Comment:
```suggestion
    public async Task CheckForUpdatesAsync()
```

nit: there should be an `Async` suffix on this method name to follow convention